baselines/compute/src/v1/addresses_client.ts.baseline

```diff
@@ -372,14 +377,22 @@ export class AddressesClient {
       'project': request.project || '',
     });
     this.initialize();
-    return this.innerApiCalls.delete(request, options, callback);
+    return this.innerApiCalls.delete(request, options, callback)
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    .then(([response, next, rawResponse]: [any, any, any]) => {
```

Comment:
It's not `next` here (it's not the pagination).  The right names would be `[response, operation, rawResponse]`. Also, we know exact types: all three have the same type `protos.google.cloud.compute.v1.IOperation` in our case :)  So we don't need to use `any` here.

Reply:
sg